src/angular/styles/_typography.scss

```diff
@@ -288,6 +329,28 @@
     --sbb-line-height: #{pxToRem($sizeLineHeightDefaultLean)};
     --sbb-line-height-medium: #{pxToRem(20)};
 
+    // Datepicker
+    --sbb-datepicker-toggle-or-arrows-padding: calc(#{pxToRem(38)} - var(--sbb-border-width));
+    --sbb-datepicker-toggle-and-arrows-padding: calc(#{pxToRem(62)} - var(--sbb-border-width));
+    --sbb-datepicker-toggle-arrow-color: var(--sbb-color-iron);
+    --sbb-datepicker-button-offset: #{pxToRem(6)};
+    // rem (or any unit) is necessary for correct calc calculation.
+    --sbb-datepicker-button-gap: 0rem;
+    --sbb-datepicker-content-offset: #{pxToRem(8)};
+    --sbb-datepicker-calender-border-color: var(--sbb-color-iron);
+    --sbb-datepicker-calender-header-height: #{pxToRem(36)};
+    --sbb-datepicker-calender-header-color: var(--sbb-color-iron);
+    --sbb-datepicker-calender-line-color: var(--sbb-color-cloud);
+    --sbb-datepicker-calender-table-content-padding-width: #{pxToRem(8)};
+    // TODO: Discuss size with jeripeierSBB
+    --sbb-datepicker-calender-arrow-width: #{pxToRem(10)};
+    --sbb-datepicker-calender-width: #{pxToRem(303)};
+    --sbb-datepicker-calender-height: #{pxToRem(303)};
+    --sbb-datepicker-calender-controls-padding: #{pxToRem(8)};
+    --sbb-datepicker-calender-body-cell-padding: #{pxToRem(16.5)};
+    --sbb-datepicker-calender-body-cell-disabled-offset: #{pxToRem(17)};
```

Comment:
```suggestion
```
should be 12 as in standard mobile definition, so therefore this definition should not be needed